codingStandards/CodingStandard-Css.md

```diff
@@ -0,0 +1,242 @@
+# CSS Coding Standard
+
+## Contents
+ - [General](#general)
+ - [CSS Files](#css-files)
+ - [Selectors](#selectors)
+   - [Selector Formatting](#selector-formatting)
+   - [Use of Efficient Selectors](#use-of-efficient-selectors)
+ - [Classes](#classes)
+   - [Naming Standards](#naming-standards)
+ - [Attributes](#attributes)
+   - [Order of Attributes](#order-of-attributes)
+   - [Attribute Formatting](#attribute-formatting)
+   - [General Details](#general-details)
+ - [Resources](#resources)
+   - [Highly Recommended](#highly-recommended)
+   - [Other Readings](#other-readings)
+
+## General
+
+* Use the [Google CSS Style Guide](https://google-styleguide.googlecode.com/svn/trunk/htmlcssguide.xml)
```

Comment:
'Google HTML/CSS Style Guide' has been moved to https://google.github.io/styleguide/htmlcssguide.xml